web-app/src/cache/users/actions.js

```diff
@@ -72,11 +73,23 @@ export const login = (token) => (
       return true
     } catch (error) {
       dispatch(removeLoggedUser())
-      await swal({
-        title: i18n.t('errorMessages.failedLogin'),
-        text: error.message,
-        icon: 'error',
-      })
+      if (error.message === upvsForbiddenSubstitutionError) {
+        await swal({
+          title: i18n.t('errorMessages.failedLogin'),
+          text: error.message,
```

Comment:
it's weird to say login failed and give user option to logout, maybe let's display in text more explaining message